releases/release-1.20/release_team.md

```diff
@@ -3,7 +3,7 @@
 | **Role** | **Name** (**GitHub / Slack ID**) | **Shadow Name(s) (GitHub / Slack ID)** |
 |---|---|---|
 | Lead | Jeremy Rickard ([@jeremyrickard](https://github.com/jeremyrickard) / Slack: `@jerickar`) | Daniel Mangum ([@hasheddan](https://github.com/hasheddan) / Slack: `@hasheddan`), Nabarun Pal ([@palnabarun](https://github.com/palnabarun) / Slack: `@palnabarun`), Savitha Raghunathan ([@savitharaghunathan](https://github.com/savitharaghunathan) / Slack: `@sraghunathan`) |
-| Enhancements | Kirsten Garrison ( [@kikisdeliveryservice](https://github.com/kikisdeliveryservice) / Slack: `@oikiki`) | |
+| Enhancements | Kirsten Garrison ( [@kikisdeliveryservice](https://github.com/kikisdeliveryservice) / Slack: `@oikiki`) | Jeremy Morris ( [@MorrisLaw](https://github.com/MorrisLaw) / Slack: `@morrislaw`), Kendall Roden ( [@kendallroden](https://github.com/kendallroden) / Slack: `@keroden`), Kinara Shah ( [@kinarashah](https://github.com/kinarashah) / Slack: `@kinara`), Mikael Johansson ( [@mikejoh](https://github.com/mikejoh) / Slack: `@mikjoh`) |
```

Comment:
nitpick: I believe we try to list shadows alphabetically by last name

Reply:
fixed - thanks!